aiven/client/cli.py

```diff
@@ -2787,12 +2806,12 @@ def service__acl_list(self):
     @arg.service_name
     @arg("--username", help="Only show rules for user", required=False)
     @arg.json
-    def service__es_acl_list(self):
+    def service__es_acl_list(self) -> None:
         """List Elasticsearch ACL configuration"""
         response = self.client.list_service_elasticsearch_acl_config(
             project=self.get_project(), service=self.args.service_name
         )
-        acl_config = response.get("elasticsearch_acl_config")
+        acl_config = response.get("elasticsearch_acl_config", {})
```

Comment:
With `acl_config = response.get("elasticsearch_acl_config")`, `acl_config` could be `None | Any`. So mypy won't allow it to be passed in `self.print_response`. And also, because it's `None | Any`, it cannot use `.get()` in the later part of the function.
And also, the `elasticsearch_acl_config` from the response will be a dictionary, so I think use default `{}` is reasonable.

Reply:
I'd convert it to `response["elasticsearch_acl_config"]` instead. Previously with `None` it would fail later on `acl_config.get()`. And with `{}` it will fail on `for acl in acl_config.get("acls")`. Both errors will be really obscure. I think it should explicitly fail if `acl_config` isn't found.